contracts/staking/compound/GoodCompoundStaking.sol

```diff
@@ -126,20 +127,22 @@ contract GoodCompoundStaking is SimpleStaking {
 		require(cToken.redeem(_amount) == 0, "Failed to redeem cToken");
 		uint256 redeemedAmount = token.balanceOf(address(this));
 		uint256 dai;
+		address recipientTemp = _recipient;
```

Comment:
seems redundant

Reply:
I added it to prevent stack too deep error